xds/internal/balancer/cdsbalancer/cdsbalancer.go

```diff
@@ -287,6 +288,7 @@ func (b *cdsBalancer) UpdateClientConnState(state balancer.ClientConnState) erro
 		}
 		b.xdsClient = c
 	}
+	b.ccw.resolverAttr = state.ResolverState.Attributes
```

Comment:
This is reading global attributes for the resolver itself.

But then you're setting it on the individual addresses.

I don't think that's correct.  I think we should propagate it down to the children in the same place, `ResolverState.Attributes`.  It's not necessarily okay to turn global or endpoint attributes into address-level attributes, as different attributes serve different purposes.